lib/rollout.rb

```diff
@@ -181,6 +215,11 @@ def active_in_group?(group, user)
     f && f.call(user)
   end
 
+  def active_in_collection?(collection, id)
+    f = collections[collection.to_sym]
```

Comment:
because the method `collections` will use `retrieve_from_storage` and will apply that hash_map treatment